tasks/tear-up.yml

```diff
@@ -4,13 +4,27 @@
   apt:
     update_cache: "yes"
 
-- name: "Install necessary packages"
+- name: "remove broken python-pip (for ubuntu 14.04)"
```

Comment:
I would like to add another pull request when we start testing with 16.04 and RHEL, and address that change at that time. There will be a bunch of those kinds of changes to make, not just here.